backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/ApiUserResolver.java

```diff
@@ -0,0 +1,33 @@
+package gov.cdc.usds.simplereport.api.apiuser;
+
+import org.springframework.stereotype.Component;
+
+import gov.cdc.usds.simplereport.api.model.User;
+import gov.cdc.usds.simplereport.db.model.ApiUser;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import gov.cdc.usds.simplereport.service.ApiUserService;
+import gov.cdc.usds.simplereport.service.OrganizationInitializingService;
+import gov.cdc.usds.simplereport.service.OrganizationService;
+import graphql.kickstart.tools.GraphQLQueryResolver;
+
+/**
+ * Created by jeremyzitomer-usds on 1/7/21
+ */
+@Component
+public class ApiUserResolver implements GraphQLQueryResolver  {
```

Comment:
Why move this to a separate class? Most of the information in the graph is about the organization (that'll change when we add roles, but only a little)—I'm not immediately seeing a benefit to extracting it.

Reply:
once i created an ApiUserMutationResolver, i wanted to follow the class structure pattern of the other entity resolvers, and looking at the whoAmI query, it felt more linked to the User "domain" than to an Organization, so I made this class and moved that query resolution here. if you feel strongly i can move it back to OrganizationResolver